pyspecProcScripts/correlation_alignment.py

```diff
@@ -170,15 +178,21 @@ def correl_align(
                     ax=ax_list[1],
                 )
                 ax_list[1].set_title("correlation function (t), \nafter apod")
+        # we want Δnu centered at 0 and don't care about the previous window
+        # selection
```

Comment:
unsure if comment wnated but this is one of the things we discussed in meeting

Reply:
I don't know what this comment means out of context.  I am reading your comments in the conversation.  If you have more detailed comments, you can add to the code.

Reply:
leave this comment.